pydra/engine/helpers_state.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, value):
         self.value = value
 
     def __str__(self):
-        return "{}".format(self.value)
+        return f"{self.value}"
```

Comment:
This seems like a weird one to use an f-string for.

```suggestion
        return str(self.value)
```